jobboard/migrations/0002_add_example_data.py

```diff
@@ -4,12 +4,14 @@
 
 class Migration(migrations.Migration):
     dependencies = [
-        ('jobboard', '0001_initial'),
+        ('jobboard', '0005_job_recruiter'),
```

Comment:
what's the reason that you add this dependency?

Reply:
I added this because I am creating recruiter objects here.